internal/bminventory/inventory.go

```diff
@@ -562,6 +565,19 @@ func (b *bareMetalInventory) RegisterCluster(ctx context.Context, params install
 			WithPayload(common.GenerateError(http.StatusInternalServerError, err))
 	}
 
+	if b.ocmClient != nil {
+		log.Infof("Creating AMS subscription for cluster %v", id)
+		sub, err := b.ocmClient.AccountsMgmt.CreateSubscription(ctx, id)
+		if err != nil {
+			return common.NewApiError(http.StatusInternalServerError, err)
+		}
```

Comment:
Why to wrap the error? why not to return the err object as it returned from AMS?

Reply:
Just a copy-past error, fixing.